dist/src/main/arctic-bin/conf/config.yaml

```diff
@@ -99,13 +99,13 @@ containers:
 #  - name: flinkContainer
 #    container-impl: com.netease.arctic.server.manager.FlinkOptimizerContainer
 #    properties:
-#      flink-home: "/opt/flink/"                                     # Flink install home
-#      target: "yarn-per-job"                                        # Flink run target, (yarn-per-job, yarn-application, kubernetes-application)
-#      export.JVM_ARGS: "-Djava.security.krb5.conf=/opt/krb5.conf"   # Flink launch jvm args, like kerberos config when ues kerberos
-#      export.HADOOP_CONF_DIR: "/etc/hadoop/conf/"                   # Hadoop config dir
-#      export.HADOOP_USER_NAME: "hadoop"                             # Hadoop user submit on yarn
-#      export.FLINK_CONF_DIR: "/opt/flink/conf/"                     # Flink config dir
+#      flink-home: /opt/flink/                                     # Flink install home
+#      target: yarn-per-job                                        # Flink run target, (yarn-per-job, yarn-application, kubernetes-application)
+#      export.JVM_ARGS: -Djava.security.krb5.conf=/opt/krb5.conf   # Flink launch jvm args, like kerberos config when ues kerberos
+#      export.HADOOP_CONF_DIR: /etc/hadoop/conf/                   # Hadoop config dir
+#      export.HADOOP_USER_NAME: hadoop                             # Hadoop user submit on yarn
+#      export.FLINK_CONF_DIR: /opt/flink/conf/                     # Flink config dir
 #      # flink kubernetes application properties.
-#      job-uri: "local:///opt/flink/usrlib/OptimizeJob.jar"          # optimizer job main jor for application mode
-#      flink-conf.kubernetes.container.image: "arctic163/optimizer-flink1.14"   #optimizer image ref
-#      flink-conf.kubernetes.service-account="flink"                 # service account to manager application job.
+#      job-uri: "local:///opt/flink/usrlib/OptimizeJob.jar"        # optimizer job main jor for application mode
+#      flink-conf.kubernetes.container.image: "arctic163/optimizer-flink1.14:latest"   #optimizer image ref
```

Comment:
The official image has change to `arctic163/optimizer-flink:0.6.0-flink1.14`  and the default job uri has change to `local:///opt/flink/usrlib/optimizer-job.jar`

Reply:
I did not find the corresponding version of the image `arctic163/optimizer-flink:0.6.0-flink1.14`. Is this the planned version number?

Reply:
Yes, because 0.6.0 has not been officially released yet. Or it can be replaced with master-snapshot. After this PR https://github.com/NetEase/amoro/pull/2136  is merged, the master branch will always automatically build the latest image.

Reply:
If you use a fixed version(like `0.6.0-flink1.14`), each version update needs to be modified synchronously.
Using `master-snapshot` will bring a version that does not match release and cause an exception.
I recommend setting it to a fixed version(like `latest`) or using a variable like `{version}` to represent it.

Reply:
Currently, we do not have a "latest" label.

Reply:
That's fine. we have tag the latest release images as `latest`